packages/block-library/src/search/edit.js

```diff
@@ -34,7 +34,7 @@ export default function SearchEdit( { className, attributes, setAttributes } ) {
 				aria-label={ __( 'Button text' ) }
 				placeholder={ __( 'Add button text…' ) }
 				keepPlaceholderOnFocus
-				formattingControls={ [] }
+				disallowFormats
```

Comment:
Why prevent the user from having formatting in the button? `<a>`, `<b>`, `<i>`, etc. are allowed inside a `<button>`.

Reply:
Changed now.